docs/markdown/snippets/build_target_older_name.md

```diff
@@ -0,0 +1,3 @@
+## Added 'name' property
+As any child of BuildTargetHolder might need the name of the object,
```

Comment:
BuildTargetHolder is the internal class name, it's not user visible. I would say something like:

Build target objects (as returned by executable(), library(), ...) now have a name() method.

Reply:
:+1: